<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="record_pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Нажмите для
записи сочитания
клавиши</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_sound_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сочитание клавиш:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="hot_key_view">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>110</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Перезаписать клавиши</string>
   </property>
  </widget>
  <zorder>hot_key_view</zorder>
  <zorder>record_pushButton</zorder>
  <zorder>name_sound_label</zorder>
  <zorder>ok_pushButton</zorder>
  <zorder>cancel_pushButton</zorder>
  <zorder>reset_pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
